src/main/java/org/apache/maven/shared/dependency/graph/DependencyGraphBuilder.java

```diff
@@ -26,7 +26,7 @@
 import java.util.Collection;
 
 /**
- * Maven project dependency graph builder API, neutral against Maven 2 or Maven 3.
+ * Maven project dependency graph builder API, based on Maven 3.1+.
```

Comment:
Wasn't this specifically the point of this API? i.e. to bridge between 2 and 3? or was that a different artifact?

Reply:
that was for maven-artifact-transfer. Yes initially that was a bridge but because now prerequisite is maven 3.1.1, no longer need old stuff  here
@rfscholte